methods/incidentReports.coffee

```diff
@@ -21,15 +21,22 @@ Meteor.methods
     return newId
 
   # similar to editIncidentReport, but allows you to set a single field without changing any other existing fields.
-  updateIncidentReport: (incident) ->
+  updateIncidentReport: (incident, fieldsToRemove=[]) ->
     _id = incident._id
     delete incident._id
     user = Meteor.user()
     if not Roles.userIsInRole(user._id, ['admin'])
       throw new Meteor.Error("auth", "User does not have permission to edit incidents")
     incident.modifiedByUserId = user._id
     incident.modifiedByUserName = user.profile.name
-    res = Incidents.update({_id: _id}, {$set: incident})
+    updateOperators =
+      $set: incident
+    if fieldsToRemove.length
+      fieldsToUnset = {}
+      fieldsToRemove.forEach (field) ->
```

Comment:
I might be looking at this the wrong way but if we combine the old and new incidents ([here](https://github.com/ecohealthalliance/eidr-connect/blob/master/client/controllers/incidentReports/incidentModal.coffee#L81)) then in the method, we won't know which is new and which should be removed?